src/main/java/org/jenkinsci/plugins/workflow/cps/CpsBodyExecution.java

```diff
@@ -116,7 +118,16 @@
 
         try {
             // TODO: handle arguments to closure
-            Object x = params.body.getBody(currentThread).call();
+            Object x;
+            try {
+                x = params.body.getBody(currentThread).call();
+            } catch (InvokerInvocationException e) {
```

Comment:
For my curiosity/learning - what's throwing the exception now that wasn't previously?